{{cookiecutter.project_slug}}/Makefile

```diff
@@ -278,7 +278,6 @@ develop: _check_setup upload-code upload-config upload-notebooks  ### Run a deve
 		--volume $(PROJECT_PATH_STORAGE)/$(CODE_DIR):$(PROJECT_PATH_ENV)/$(CODE_DIR):rw \
 		--volume $(PROJECT_PATH_STORAGE)/$(CONFIG_DIR):$(PROJECT_PATH_ENV)/$(CONFIG_DIR):ro \
 		--volume $(RESULTS_DIR_STORAGE):$(PROJECT_PATH_ENV)/$(RESULTS_DIR):rw \
-		--env PYTHONPATH=$(PROJECT_PATH_ENV) \
```

Comment:
it's better to find a better solution and not loose this functionality, see https://github.com/neuromation/platform-client-python/issues/1316

Reply:
Fixed so that it does not loose functionality, and works both on Windows (Anaconda Prompt, from Makefile it's expanded to `--env PYTHONPATH=/project`) and on Linux (`--env PYTHONPATH=//project` which also works well since `//project` is the same as `/project`)